_posts/2020-08-20-teaching-code-as-a-foreign-language.md

```diff
@@ -14,14 +14,9 @@ sponsors:
 tags:
   - teaching
   - coaching
-venue:
-  name: The Skiff
-  link: https://www.theskiff.org/
-  location: https://goo.gl/maps/SPf97i5WkAR2
-  address: Ironworks, 30 Cheapside, Brighton, BN1 4GD
-  latlong: 50.829371,-0.138194
 layout: event.html
 collection: events
+online: true
```

Comment:
Cool - is this metadata used anywhere?

Reply:
didn't check tbh, just noticed it on the other online events

Reply:
ah yep, it's used in `_includes/partials/eventDetail.html`

Reply:
Yeah just saw, awesome